contrib/sbtc/token-key-admin/contracts/sbtc-token-key-admin.clar

```diff
@@ -0,0 +1,115 @@
+
+;; title: sbtc-token-key-admin
+;; version:
+;; summary:
+;; description:
+
+;; traits
+;;
+(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait-ft-standard.sip-010-trait)
+
+;; token definitions
+;; 
+(define-fungible-token sbtc)
+
+;; constants
+;;
+(define-constant contract-owner tx-sender)
+(define-constant err-invalid-caller (err u1))
+(define-constant err-owner-only (err u100))
+(define-constant err-not-token-owner (err u101))
+
+;; data vars
+;;
+(define-data-var coordinator (optional principal) none)
```

Comment:
Would it make sense to just initialize `coordinator` to be the intended principal from the get-go, instead of requiring the coordinator to call `set-coordinator-key`?

Reply:
We were assuming that the principal deploying the contract would be an admin, but not necessarily the coordinator.  And since we are going to need separate FROST public keys in addition to principals, I'm not sure there's a good way to set that on contract init.

Reply:
Can we initialize coordinator to the contract admin/creator? The admin can later change it and this way the variable doesn't need to be optional.

Reply:
> Can we initialize coordinator to the contract admin/creator? The admin can later change it and this way the variable doesn't need to be optional.

No, because we need a FROST public key in addition to a principal.